src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2669,10 +2709,16 @@ void InitSolarHeatGains(EnergyPlusData &state)
             Array1D<Real64>::size_type lSP = state.dataSurface->SurfReflFacBmToBmSolObs.index(state.dataGlobal->PreviousHour, 1) - 1;
             // For Complex Fenestrations:
             for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
+
+                Real64 GndSurfReflectance = 0.0;
+                if (state.dataSurface->UseSurfPropertyGndSurfRefl(SurfNum)) {
+                    GndSurfReflectance = state.dataSurface->GroundSurfsProperty(state.dataSurface->GroundSurfsPropertyNum(SurfNum)).SurfsReflAvg;
+                } else {
+                    GndSurfReflectance = state.dataEnvrn->GndReflectance;
+                }
```

Comment:
Ground reflectance value from the new Ground surfaces object `SurfaceProperty:GroundSurfaces` is used if specified, instead of the default global.